ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vlan-interfaces.md

```diff
@@ -125,20 +127,27 @@ interface Management1
 
 #### IPv6
 
-| Interface | VRF | IPv6 Address | Virtual Router Address | VRRP | ND RA Disabled | Managed Config Flag | IPv6 ACL In | IPv6 ACL Out |
+| Interface | VRF | IPv6 Address | IPv6 Address Virtual | Virtual Router Address | VRRP | ND RA Disabled | Managed Config Flag | IPv6 ACL In | IPv6 ACL Out |
```

Comment:
Did you add these entries manually? I don't see any change in jinja template for documentation that renders this configuration.

Reply:
I have indeed added them manually on the documentation, wasn't sure whether it was auto generated or not.

As for the jinja template that renders that configuration: 
eos_cli_config_gen/templates/eos/vlan-interfaces.j2  [LINE 66]

Reply:
So, in the check details, it seems that there's a conflict indeed in the documentation regarding vlans, due the the field that I have added. 

Should I revert the documentation back?

Reply:
It is auto-generated. There should be a jinja template in the documentation folder.

Reply:
Check this jinja template. Make changes to this template: https://github.com/aristanetworks/ansible-avd/blob/bf99bca2dbed1c70f09bfd11b23d1296c3e5e7b2/ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/vlan-interfaces.j2

Reply:
Also you need to run molecule to test. Check this page https://avd.sh/en/releases-v3.x.x/molecule/index.html#ansible-molecule

Reply:
Let me know if you can do it. Else I will take this PR as I was already working on this.

Reply:
Could you please solve it? I'm really not that familiar with molecule, same with the documentation, I'm not sure what to define there exactly.

Reply:
Ok, I will take this PR.